mypy_django_plugin/lib/fullnames.py

```diff
@@ -37,7 +37,6 @@
         FOREIGN_OBJECT_FULLNAME,
         FOREIGN_KEY_FULLNAME,
         ONETOONE_FIELD_FULLNAME,
-        MANYTOMANY_FIELD_FULLNAME,
```

Comment:
For extra clarification: the removal of this line detaches `ManyToManyField` to be handled by the `_GT` and `_ST` generic arguments and the attributes `_pyi_private_set_type` and `_pyi_private_get_type`